interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
Don't understand this one because I believe we are echoing to the screen the pharmacies as they are being inserted and as I type this it seems you want this changed to a waiting image until the job is complete. Am I understanding that correctly?